chalicelib/fs_connection.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, fs_environ, fs_environ_info, test=False, use_es=True):
         self.ff_es = fs_environ_info['es']
         if not test:
             self.ff_s3 = s3Utils(env=self.ff_env)
-            try:
+            try:  # TODO: make this configurable from env variables?
```

Comment:
Env variables are useful for bootstrapping the system, but aren't those variable supposed to get shuttled into the registry for use in the application? It's possible that security keys should get copied into aws credential files and not the registry...